scripts/install.sh

```diff
@@ -114,8 +115,8 @@ export DATABASE_URL=postgresql://$APP_DB_USER:$APP_DB_PASS@localhost:5432/$APP_D
 cd /vagrant
 #Flask
 echo "Installing requirements"
+pip install future
 pip install -r requirements/dev.txt
 
-python create_db.py
```

Comment:
>Remove the execution of python create_db.py to match with the setup instructions.

If anything, we should change the instructions to match this line, the idea is to speed up the deployment of the site, not to slow it down, the less manual work, the better.

Reply:
One of the reason why it made sense to me to remove this statement from `install.sh` script is because this script requires user input for getting the email address and password and this can be done from the install.sh script for vagrant setup either by running the script manually or by picking up this values from environment variable which seems too be a very tedious task.

Reply:
should we set some default email and password values?

Reply:
Anyway, I just realized that run.sh has the create_db.py as well.